gradle-plugin/configure/build.gradle.kts

```diff
@@ -1,28 +1,28 @@
-import com.novoda.gradle.release.PublishExtension
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 /// The plugin version number – change this to match whatever your tag will be
-version = "0.5.0"
 group = "com.automattic.android"
+version = "0.6.0"
```

Comment:
**Out of scope**, but I wonder if we could do something similar to [Ruby's `version.rb`](https://github.com/wordpress-mobile/release-toolkit/blob/develop/lib/fastlane/plugin/wpmreleasetoolkit/version.rb) to DRY this definition. 

I don't think TOML supports reading from files, but Kotlin scripts should be able to read a `.toml` and extract the version.

cc @jkmassel

Reply:
@jkmassel mentioned that he looked into this, but at least the initial attempt did not work out. What we are hoping to do is to package the binary together with the gradle plugin. We'll also remove the `version` from the Gradle plugin and use our naming scheme of `develop-{sha1}`, `{tagName}` etc, so that should help with the issue.

Unfortunately these improvements are currently in the back burner while we address a few high priority issues in other projects.

Reply:
Fair. Thank you for letting me know.